traitsui/theme.py

```diff
@@ -66,45 +65,48 @@ def __init__(self, image=None, **traits):
     # -- Property Implementations ---------------------------------------------
 
     def _get_content_color(self):
-        import wx
+        if ETSConfig.toolkit == "wx":
+            import wx
 
-        if self._content_color is None:
-            color = wx.BLACK
-            islice = self.image_slice
-            if islice is not None:
-                color = islice.content_color
+            if self._content_color is None:
+                color = wx.BLACK
+                islice = self.image_slice
+                if islice is not None:
+                    color = islice.content_color
 
-            self._content_color = color
+                self._content_color = color
 
-        return self._content_color
+            return self._content_color
```

Comment:
This `return self._content_color` should be dedented so that even if wx is not available, the value given by the setter can be retrieved via the getter.  Maybe worth adding a test for this?